README.md

```diff
@@ -139,6 +139,10 @@ and speed in our CI system.
 - **[MacStadium][23]**: Managed hosting provider for Mac. Provides Mac
   hardware used for testing in the Node.js project's CI system.
 
+- **[Packet][24]**: Bare metal cloud for developers. Through their
+  [Works on Arm partnership](https://www.worksonarm.com/),
+  Packet provides Armv8 build infrastructure.
```

Comment:
Ugh, this poor capitalisation is killing me but I see we're guilty of inheriting from their wording. My preference would be "Works on ARM" and "ARMv8", and "ARM64" would be even better here since that's becoming the canonical name for it. What do you think @mhdawson?

Reply:
I think we probably need to stick to the name they chose for "Works on Arm partnership", but I think its ok change the capitalization to "Works on arm partnership" and then later say "Packet provides ARM64 build infrastructure" if you think that is more accurate.